setup.cfg

```diff
@@ -3,3 +3,4 @@ test=pytest
 
 [flake8]
 max-line-length=100
+ignore=D102,D100,D103,D104,E211
```

Comment:
I got the same errors locally. Did you install flake8-docstrings? Without that installed this class of errors won't show up.